### ISSUE TITLE
Wrong in setting up new database

### ISSUE THREAD

**Issue by xiaonian92** [sqjin/CellChat#265]
Hi dear author, I have successfully set up a new database for pig by following your instructions in issue#38. However, when I run `cellchat <- subsetData(cellchat)`, I got :`Issue identified!! Please check the offical Gene Symbol of the following genes: 469, 470, 471, 29, 30 ... .... `And if I omitted this warning, I got very few df.net.
Could you please give me some advice? Thank you so much!

**Comment by sqjin** [comment]
Clearly, 469, 470, ... are not the gene names. If you can share me your new database, I can check whether you correctly format the related files.

**Comment by xiaonian92** [comment]
> 
> 
> Clearly, 469, 470, ... are not the gene names. If you can share me your new database, I can check whether you correctly format the related files.

Thanks! But I wonder how could I share the database to you? I tried `usethis::use_data(CellChat.pig, overwrite=TRUE)` but got error: `Path does not appear to be inside a project or package`. Now I'm trying to upload the four files to my own Giuhub repositories, I'm not good at doing this and I hope it will work. Is there other ways?

**Comment by sqjin** [comment]
You can share me the files by sending email to `cellchat.package@gmail.com`. The `usethis::use_data(CellChat.pig, overwrite=TRUE)` can be done if you re-build CellChat package on your local computer. You can check how to build a R package.

**Comment by xiaonian92** [comment]
> 
> 
> You can share me the files by sending email to `cellchat.package@gmail.com`. The `usethis::use_data(CellChat.pig, overwrite=TRUE)` can be done if you re-build CellChat package on your local computer. You can check how to build a R package.

Hi, I already sent all files and code to this email address. Thanks!

**Comment by xiaonian92** [comment]
> 
> 
> You can share me the files by sending email to `cellchat.package@gmail.com`. The `usethis::use_data(CellChat.pig, overwrite=TRUE)` can be done if you re-build CellChat package on your local computer. You can check how to build a R package.

Hi, I already sent all files and code to this email address. And I also upload files to my repository [cellchat.pig](https://github.com/xiaonian92/cellchat.pig). Thank you for your kindly reply!

**Comment by xiaonian92** [comment]
Hello sqjin, good news! I have figured this error out. I checked cellchat.human and my database, and found there're some differences of the dataset type, and I tried to convert 'factor' type to 'charactor' (with lots of other troubles of course), finally I made it! Though I still got `issue identified!!`, but it only contains 8 aberrant genes, I decided to ignore them. 
Thanks very much, anyway. Wish you a good day~
